examples/project_grid.py

```diff
@@ -0,0 +1,78 @@
+"""
+Projection of gridded data
+==========================
+
+Sometimes gridded data products need to be projected before they can be used.
+For example, you might want to project the topography of Antarctica from
+geographic latitude and longitude to a planar polar stereographic projection
+before doing your analysis. When projecting, the data points will likely not
+fall on a regular grid anymore and must be interpolated (re-sampled) onto a new
+grid.
+
+Function :func:`verde.project_grid` automates this process using the
+interpolation methods available in Verde. An input grid
+(:class:`xarray.DataArray`) is interpolated onto a new grid in the given
+`pyproj <https://jswhit.github.io/pyproj/>`__ projection. The function takes
+care of choosing a default grid spacing and region, running a blocked mean to
+avoid spatial aliasing (using :class:`~verde.BlockReduce`), and masking the
+points in the new grid that aren't constrained by the original data (using
+:func:`~verde.convexhull_mask`).
```

Comment:
This is a great description of how it works and how the function deals with the original data. I dig it!

Reply:
Thanks! I did mostly because I had to dig into the GMT code in order to see what they did. It's fancier than this but Verde is much easier to install :)